docker/.env.example

```diff
@@ -1,4 +1,7 @@
 COMPOSE_PROJECT_NAME=demo
+# Enable this line to start the backend and frontend containers
+# COMPOSE_PROFILES=backend,frontend
```

Comment:
Let's enable them by default because only devs won't need them. So please uncomment the line :)